Tasks/PublishTestResultsV2/package.json

```diff
@@ -17,9 +17,13 @@
   },
   "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
   "dependencies": {
-    "@types/node": "^6.0.101",
-    "@types/q": "^1.0.7",
-    "azure-pipelines-task-lib": "2.8.0",
-    "uuid": "^3.0.1"
+    "@types/node": "^10.17.0",
+    "@types/q": "^1.5.0",
+    "@types/mocha": "^5.2.7",
```

Comment:
Switch to Mocha.Done instead of MochaDone in tests

Reply:
Fixed